ci/docker-push.yml

```diff
@@ -0,0 +1,71 @@
+name: Docker
+
+on:
+  push:
+    # Publish `master` as Docker `latest` image.
+    branches:
+      - master
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+
+  # Run tests for any PRs.
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
There didn't seem to be a way to accept the default types for `pull_request`. Was hoping something like `pull_request: true` would have done this.

Reply:
IIRC, just using `pull_request:` will do what you're asking here. 😸

Reply:
@anthonysterling thanks! That worked. Should be resolved with pushed changes.